lib/GILGen/GILGenStmt.cpp

```diff
@@ -101,19 +106,15 @@ struct GILGenStmt : public ASTVisitor<GILGenStmt, void> {
 
     void visitBreakStmt([[maybe_unused]] BreakStmt *stmt)
     {
-        Scope *scope = &getCurrentScope();
-        while (scope && !scope->isLoopScope())
-            scope = scope->parent;
+        Scope *scope = dropLoopScopes();
         assert(scope && "Break statement outside of loop");
```

Comment:
if you really wanted to not duplicate the assert, you could put it at the end of `dropLoopScopes`, that would make sense